src/sora-economy.md

```diff
@@ -116,7 +116,7 @@ XOR supply in the SORA mainnet is backed by acceptable liquid cryptocurrencies,
 - XST
 - TBCD
 
-The token bonding curve is built directly into [Polkaswap.io](https://polkaswap.io) such that if the secondary market price of XOR is exceeded for one of the above trading pairs, the token bonding curve will automatically be executed so that new XOR will be put into circulation, 20% of the buying currency will be put into a pool to buy back and burn XOR, and 80% of the buying currency will go to reserves, held by the token bonding curve. As reserves are in multiple currencies, future releases of the system should provide for ways to balance between the currency reserves to provide liquidity in a variety of tokens. The close integration of the SORA economic model with Polkaswap will allow this process to be automated at some point in the future.
+The token bonding curve is built directly into [Polkaswap.io](https://polkaswap.io) such that if the secondary market price of XOR is exceeded for one of the above trading pairs, the token bonding curve will automatically be executed so that new XOR will be put into circulation, 20% of the buying currency will be put into a pool to buyback-and-burn XOR, and 80% of the buying currency will go to reserves, held by the token bonding curve. As reserves are in multiple currencies, future releases of the system should provide for ways to balance between the currency reserves to provide liquidity in a variety of tokens. The close integration of the SORA economic model with Polkaswap will allow this process to be automated at some point in the future.
```

Comment:
```suggestion
The token bonding curve is built directly into [Polkaswap.io](https://polkaswap.io) such that if the secondary market price of XOR is exceeded for one of the above trading pairs, the token bonding curve will automatically be executed so that new XOR will be put into circulation, 20% of the buying currency will be put into a pool to buyback-and-burn XOR, and 80% of the buying currency will go to reserves, held by the token bonding curve. As reserves are in multiple currencies, future releases of the system should provide ways to balance between the currency reserves to provide liquidity in various tokens. The close integration of the SORA economic model with Polkaswap will allow this process to be automated at some point in the future.
```